content/work/cfoshare.md

```diff
@@ -20,18 +21,68 @@ press:
     venue: *client
 links:
   site: https://cfoshare.org/
+sector:
+  - Finance
+  - Accounting
+tags:
+  - Finance
+  - Accounting
```

Comment:
If we end up using the same names, and we prefix the tags - we could probably auto-generate sector names from tag names… (I guess we'll see where all that lands)

Reply:
@mirisuzanne @dvdherron @stacyk 

- If we could change the date macro to allow display of only the years for client projects, I think that would be ideal. Any other date info seems not as useful to visitors, and looking back through all the projects to find accurate start/end months would take some doing. (I'm willing to hear arguments to the contrary, though.)
- It would be nice if we could just put in tags and auto-generate sector names. Or maybe we remove the sector information from the sub-header and have the tags say "Accounting Sector" and Financial Sector."